tests/errors/ParseBind.hs

```diff
@@ -1,3 +1,4 @@
+-- Fails because we cannot catch parse errors with --expect-error-containing
 {-@ LIQUID "--expect-error-containing=Cannot parse specification" @-}
 -- We should reject the below to disallow uppercase binders
 
```

Comment:
Should I fix this?

Reply:
Why is that we can't filter parse errors?

Reply:
A parse error is thrown while parsing the `{-@ @-}` specifications.  So, the `--expect-error-containing` flag is not available when the error is thrown.

Reply:
We could separate and digest the liquid comments first `{-@ LIQUID ... @-}` ahead of parsing the rest of the comments. Could be something for another PR.

Reply:
[7de09d](https://github.com/ucsd-progsys/liquidhaskell/pull/1984/commits/47de09d7b80b66d8ab8006842a546414bccce564) has some similar work that I had to do to catch a bunch of other errors, though what you're suggesting would be slightly more involved, but possible.

Reply:
An alternative might be moving those tests to a `parsing-errors` test group, and providing a uniform `--fplugin-opt=LiquidHaskell:--expect-error-containing="Parse"` flag in the cabal file, or similar.  This would provide it without needing to parse the spec.